zeithistorische-forschungen.csl

```diff
@@ -0,0 +1,393 @@
+  </macro>
+  <macro name="date">
+    <choose>
+      <if variable="issued">
+        <date date-parts="year" form="numeric" variable="issued"/>
+      </if>
+      <else>
+        <text term="no date"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="issued">
+    <group prefix=", " delimiter=", ">
+      <group delimiter=" ">
+        <text variable="genre"/>
+        <choose>
+          <if type="report">
+            <text variable="number"/>
+          </if>
+        </choose>
+      </group>
+      <text variable="event"/>
+      <text macro="collection"/>
+      <text macro="edition"/>
+    </group>
+    <choose>
+      <if type="book chapter thesis" match="any">
+        <group prefix=", " delimiter=" ">
+          <text macro="published"/>
+          <text macro="date"/>
+        </group>
+      </if>
+      <else-if type="article-journal article-magazine" match="any">
+        <group prefix=" " delimiter=" ">
+          <group delimiter=", ">
+            <text macro="volume"/>
+            <text macro="secondary-container-contributors"/>
+          </group>
+          <text prefix="(" macro="date" suffix=")"/>
+        </group>
+      </else-if>
+      <else>
+        <group prefix=", " delimiter=", ">
+          <text macro="published"/>
+          <text macro="date"/>
+        </group>
+      </else>
+    </choose>
+    <choose>
+      <if type="manuscript">
+        <text prefix=" (" value="Typoskript" suffix=")"/>
+      </if>
+    </choose>
+  </macro>
+  <macro name="pages">
+    <choose>
+      <if type="article-journal article-magazine article-newspaper chapter" match="any">
+        <group delimiter=". ">
+          <label variable="page" form="short"/>
+          <text variable="page"/>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <macro name="access">
+    <group delimiter=", ">
+      <choose>
+        <if type="webpage" match="any">
+          <date variable="accessed" form="numeric"/>
+        </if>
+      </choose>
+      <text variable="URL" prefix="URL: &lt;" suffix="&gt;"/>
+    </group>
+  </macro>
+  <macro name="citation-locator">
+    <group delimiter=". ">
+      <label variable="locator" form="short"/>
+      <text variable="locator"/>
+    </group>
+  </macro>
+  <citation et-al-min="4" et-al-use-first="1" disambiguate-add-names="true">
+    <layout suffix="." delimiter="; ">
+      <choose>
+        <if position="ibid-with-locator">
+          <text term="ibid" text-case="capitalize-first" suffix="."/>
+          <text prefix=", " macro="citation-locator"/>
+        </if>
+        <else-if position="ibid">
+          <text term="ibid" text-case="capitalize-first" suffix="."/>
+        </else-if>
+        <else-if position="subsequent">
+          <group delimiter=", ">
+            <group delimiter=", ">
+              <text macro="contributors-short"/>
+              <text macro="title-short"/>
+            </group>
+            <text macro="citation-locator"/>
+          </group>
```

Comment:
In der Vorgabe sehe ich hier noch einen Querverweis (Anm. 12). Wenn das gewollt ist lässt sich das mit `first-reference-note-number` machen